server/src/main/java/org/elasticsearch/index/shard/IndexShard.java

```diff
@@ -1432,6 +1433,15 @@ public DenseVectorStats denseVectorStats() {
         return getEngine().denseVectorStats();
     }
 
+    public SparseVectorStats sparseVectorStats() {
```

Comment:
I don't think we need to create specific sparse_vector stats in order to get mapping stats; we already have this count in cluster stats and should be ready in telemetry indices as well.

Running _cluster/stats already provides:

```json
{
  "indices": {
    "mappings": {
      "total_field_count": 38,
      "total_deduplicated_field_count": 38,
      "total_deduplicated_mapping_size_in_bytes": 1878,
      "field_types": [
        {
          "name": "sparse_vector",
          "count": 1,
          "index_count": 1,
          "script_count": 0
        }
      ]
    }
  }
}
```

If there's nothing else we need to add, we probably don't need to explicitly collect stats.

Reply:
Thanks. If this is all we can get maybe our job is easier @Mikep86 😅 

Is there anything else we can get surrounding usage or dimensions of sparse_vector fields though?

Reply:
Always nice to see things get simpler 😃